akka-samples/akka-sample-distributed-data-java/tutorial/index.html

```diff
@@ -0,0 +1,293 @@
+<!-- <html> -->
+<head>
+<title>Akka Distributed Data Samples with Java</title>
+</head>
+
+<body>
+
+<div>
+<p>
+This tutorial contains 5 samples illustrating how to use  
+<a href="http://doc.akka.io/docs/akka/2.4-SNAPSHOT/java/distributed-data.html" target="_blank">Akka Distributed Data</a>.
+</p>
+<ul>
+<li>Low Latency Voting Service</li>
+<li>Highly Available Shopping Cart</li>
+<li>Distributed Service Registry</li>
+<li>Replicated Cache</li>
+<li>Replicated Metrics</li>
+</ul>
+
+<p>
+<b>Akka Distributed Data</b> is useful when you need to share data between nodes in an
+Akka Cluster. The data is accessed with an actor providing a key-value store like API.
+The keys are unique identifiers with type information of the data values. The values 
+are <i>Conflict Free Replicated Data Types</i> (CRDTs).
+</p>
+
+<p>
+All data entries are spread to all nodes, or nodes with a certain role, in the cluster
+via direct replication and gossip based dissemination. You have fine grained control
+of the consistency level for reads and writes.
+</p>
+
+<p>
+The nature CRDTs makes it possible to perform updates from any node without coordination.
+Concurrent updates from different nodes will automatically be resolved by the monotonic
+merge function, which all data types must provide. The state changes always converge.
+Several useful data types for counters, sets, maps and registers are provided and
+you can also implement your own custom data types.
+</p>
+
+<p>
+It is eventually consistent and geared toward providing high read and write availability
+(partition tolerance), with low latency. Note that in an eventually consistent system a read may return an 
+out-of-date value.
+</p>
```

Comment:
yes, perhaps a link to http://doc.akka.io/docs/akka/2.4-M2/scala/distributed-data.html#Limitations